python/tk_multi_launchapp/base_launcher.py

```diff
@@ -88,12 +90,15 @@ def _register_launch_command(
             "shotgun_version",
         ]
         if self._tk_app.engine.environment.get("name") not in skip_environments:
-            properties = {
+            if not properties:
```

Comment:
minor: we normally use the std pattern `properties = properties or {}` for these cases